api/server/routes.go

```diff
@@ -31,5 +31,10 @@ func (c *clusterApi) Routes() []*Route {
 		{verb: "POST", path: clusterPath(client.UriNode, cluster.APIVersion), fn: c.setNodeConf},
 		{verb: "DELETE", path: clusterPath(client.UriNode+"/{id}", cluster.APIVersion), fn: c.delNodeConf},
 		{verb: "GET", path: clusterPath("/getnodeidfromip/{idip}", cluster.APIVersion), fn: c.getNodeIdFromIp},
+		{verb: "GET", path: clusterVersion("checksecretlogin", cluster.APIVersion), fn: c.secretLoginCheck},
```

Comment:
Since these are all secret REST command, place them all under `/secret`. Make a new function called `clusterSecretPath("checklogin", cluster.APIVersion)` where it returns `/v1/cluster/secret/checklogin`. Then do the same for the others.

Reply:
ok, also should I make them more REST like.
e.g instead of /setclustersecretkey , POST call on /clustersecretkey,
similarly /checksecretlogin,- > GET on /secretlogin -> allow to check secrets  session is vaild.
and POST on /secretlogin will allow you to store secrets in secrets store.

Note: Earlier one was same as PX's